pyanaconda/modules/payloads/payload/rpm_ostree/installation.py

```diff
@@ -496,9 +498,33 @@ def name(self):
         return "Configure OSTree bootloader"
 
     def run(self):
-        self._move_grub_config()
+        if have_bootupd(self._sysroot):
+            self._install_bootupd()
+        else:
+            self._move_grub_config()
         self._set_kargs()
 
+    def _install_bootupd(self):
+        bootloader = STORAGE.get_proxy(BOOTLOADER)
+
+        rc = execWithRedirect(
+            "bootupctl",
+            [
+                "backend",
+                "install",
+                "--auto",
+                "--with-static-configs",
+                "--device",
+                "/dev/{}".format(bootloader.Drive),
```

Comment:
Hmm, I am not sure if this will work for all types of devices. I think you should call `GetDeviceData` and use the `path` attribute.